File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        String astr = String.valueOf(a);
        String bstr = String.valueOf(b);
        
        String ab = astr + bstr;
        String ba = bstr + astr;
        
        int abInt = Integer.parseInt(ab);
        int baInt = Integer.parseInt(ba);
        
        if (abInt > baInt || abInt == baInt) {
            answer = abInt;
        }
        else if (abInt < baInt) {
            answer = baInt;
        }
        
        
        return answer;
    }
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.py
def solution(a, b):
    answer = 0
    ab = ''
    ba = ''
    ab = str(a) + str(b)
    ba = str(b) + str(a)
    
    if int(ba) > int(ab) :
        answer = int(ba)
    elif int(ab) > int(ba) :
        answer = int(ab)
    elif int(ab) == int(ba) :
        answer = int(ab)
        
    return answer